crates/rust-analyzer/src/world.rs

```diff
@@ -159,20 +176,7 @@ impl WorldState {
         change.set_crate_graph(crate_graph);
 
         // FIXME: Figure out the multi-workspace situation
```

Comment:
Have we already works in multi-workspace ?

Reply:
Indeed, thanks, I'll remove this fixme a bit later.